parkhangs-webapp/src/features/modal/ModalEventDetail.js

```diff
@@ -161,6 +198,9 @@ class ModalEventDetail extends React.Component {
                                 `${this.state.currentTab === 'comments' ? 'tab-selected' : ''} ${'ModalEventDetail-tabSelector'}`}
                                 onClick={() => this.selectTab('comments')}>
                                 <span className="ModalEventDetail-tabName">Comments</span>
+                                <div>
```

Comment:
👋

Reply:
if you need space between elements, use css please

Reply:
whoops that extra div was there for no reason haha,; removed